File: R/linear_search.R
# Linear Search in R

linear_search <- function(vec, num){
	for (i in vec){
		if (i == num)
			return(TRUE)
	}
	return(FALSE)
}

ans = linear_search( c(1:10), 7 )

ifelse(ans == TRUE, "Number found in vector", "Number not found in vector")

File: php/linear_search.php
<html>
<head>
<title>linear search</title>
</head>
<body>
<?php 

    function linear_search($arr,$k)
    {
        $n=count($arr);
        for($i=0;$i<$n;$i++)
        {
            if($arr[$i]==$k)
            {
                return $i+1;
            }
        }
        return -1;
    }

?>
</body>
</html>